cpp/src/centrality/betweenness_centrality_impl.cuh

```diff
@@ -39,6 +42,15 @@
 #include <cuda/std/optional>
 #include <thrust/functional.h>
 #include <thrust/reduce.h>
+#include <thrust/sort.h>
+#include <thrust/transform.h>
```

Comment:
We no longer call thrust::transform, and I assume this include statement is no longer necessary.